ui/utils_ui.py

```diff
@@ -7,7 +7,7 @@ def base_poll(context):
     return context.active_object and context.active_object.type == "MESH"
 
 
-def draw_lp_group(layout, ntree, group_node, inp_offset = 1):
+def draw_lp_group(layout, ntree, group_node, mat, layer, inp_offset = 1):
```

Comment:
What's the reason you're passing in the material and layer here? Isn't this always the active material and layer?

Reply:
Ok, will merge then